ngrinder-frontend/src/js/components/perftest/detail/Detail.vue

```diff
@@ -160,7 +163,7 @@
                 progressMessage: test.progressMessage,
                 lastProgressMessage: test.lastProgressMessage,
                 testName: test.testName,
-                tagString: test.tagString,
+                tagList: test.tagString !== '' ? test.tagString.split(',') : [],
```

Comment:
`!== ''` seems unnecessary code 

`test.tagString ? test.tagString.split(',') : []`

Reply:
Amended it. Thanks!